blog/content/post/2021-02-05-testing.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Manual vs Automated Testing
+abstract: 'Pros and cons (if there are any) of automated testing and the process of transparent validation.'
+cover: /covers/test-cover.png
+author:
+  - piotr.konkol
+  - kamila.banecka
+layout: post
+published: true
+date: 2021-02-05
+archives: "2021"
+
+tags:
+  - testing
+  - firmware
+categories:
+  - Firmware
```

Comment:
@KBanecka do we plan to use all categories for this post?

Reply:
additrional categories removed